chemreg/substance/tests/test_views.py

```diff
@@ -578,3 +579,58 @@ def test_intermediate_user_post(client, admin_user, synonym_type_factory):
         {"data": {"type": "synonymType", "attributes": stf.initial_data}},
     )
     assert resp.status_code == 201
+
+
+def test_substance_relationship_view():
+    """Tests that the Substance Relationship View Set, is a sub class of the Model View Set."""
+    assert issubclass(SubstanceRelationshipViewSet, ModelViewSet)
+
+
+@pytest.mark.django_db
+def test_substance_relationship_list(
+    client, admin_user, substance_relationship_factory
+):
+    client.force_authenticate(user=admin_user)
+    substance_relationship_factory.create()
+    resp = client.get("/synonyms")
```

Comment:
This test appears to be hitting the wrong endpoint
```suggestion
    resp = client.get("/substanceRelationships")
```